crates/sui-json/src/lib.rs

```diff
@@ -818,6 +819,27 @@ fn resolve_call_arg(
     }
 }
 
+pub fn is_receiving_argument(view: &BinaryIndexedView, sig_tok: &SignatureToken) -> bool {
```

Comment:
`sig_tok` is named `arg_type` [below](https://github.com/MystenLabs/sui/pull/12988/files#diff-7fdf6661cb291534afd772cb58c627ba2d2c35e3ee84d9f04eb45bcb3111f8c7R331), and I was wondering if there's a specific reason as to not name them the same.

Reply:
Not really -- happy to make them agree :)